content/blog/2022-05-19-A_Type_and_Effect_System_for_Deterministic_Parallel_Java/index.md

```diff
@@ -0,0 +1,123 @@
++++
+title = "A Type and Effect System for Deterministic Parallel Java"
+[extra]
+bio = """
+  [Nikita Lazarev](https://www.nikita.tech/) is an PhD student at Cornell's ECE department;
+  Susan Garry is an undergraduate student at Cornell's CS department.
+"""
+[[extra.authors]]
+name = "Nikita Lazarev"
+link = "https://www.nikita.tech/"  # Links are optional.
+[[extra.authors]]
+name = " Susan Garry"
++++
+
+### Paper review
```

Comment:
I don't think you need this heading. (People can tell it's a paper review by reading the first bit.)